packages/apps/reputation-oracle/server/src/modules/escrow-completion/escrow-completion.service.ts

```diff
@@ -116,21 +127,49 @@ export class EscrowCompletionService {
             );
           }
 
-          await this.payoutService.executePayouts(
+          const calculatedPayouts = await this.payoutService.calculatePayouts(
             escrowCompletionEntity.chainId,
             escrowCompletionEntity.escrowAddress,
             escrowCompletionEntity.finalResultsUrl,
-            escrowCompletionEntity.finalResultsHash,
+          );
+
+          /**
+           * When creating payout batches we need to guarantee deterministic result,
+           * so order it first.
+           */
+          const payoutBatches = _.chunk(
+            _.orderBy(calculatedPayouts, 'address', 'asc'),
+            ESCROW_BULK_MAX_COUNT,
```

Comment:
According to the contract pay outs length must be less than 100. So ESCROW_BULK_MAX_COUNT should 99

Reply:
Agree, but a contract upgrade means a long process for such a small change. Let's have it mind for future updates.